src/App.tsx

```diff
@@ -1,22 +1,41 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const isPropertyMatchQuery = (movieProperty: string) => (
+    movieProperty
+      .trim()
+      .toLocaleLowerCase()
+      .includes(query.toLocaleLowerCase().trim())
```

Comment:
Use 'toLowerCase()' instead of 'toLocaleLowerCase()' for better performance, since we don't need locale-specific comparison.